### ISSUE TITLE
Packages with vulnerabilities

### ISSUE THREAD

**Issue by jvsancho80** [Fody/Virtuosity#601]
There are 2 vulnerabilities when use de lastest version 3.1.1

![image](https://github.com/user-attachments/assets/2f1c65d8-32cf-4f89-a428-b17c601cd86e)

![image](https://github.com/user-attachments/assets/ce8cfc86-9344-412a-93d0-51c959bfac1d)

![image](https://github.com/user-attachments/assets/e1451022-3d48-4366-8b2f-18ddbecfb646)

**Comment by tom-englert** [comment]
These are just reference assemblies of netstandard.
They will resolve to the latest versions of the installed framework on your machine at runtime.

So this is only an issue if you are running netcore 3.0 or earlier

**Comment by jvsancho80** [comment]
I use Net 8

**Comment by tom-englert** [comment]
So you're safe

**Comment by ltrzesniewski** [comment]
We could make a new release, that would drop the netstandard1.0 target and upgrade it to netstandard2.0 so nuget would be happy.

**Comment by tom-englert** [comment]
But it's already 2.0

**Comment by ltrzesniewski** [comment]
Yeah, in code, but the release dates from 2021 and it's 1.0 there.

**Comment by tom-englert** [comment]
@jvsancho80: 3.1.2 has the latest dependencies

**Comment by jvsancho80** [comment]
Ok. Thanks.
